### PR TITLE
Made progress on bootstrap project

### PR DESCRIPTION
**Pull request by mockitoguy**
It's coming along! Bootstrap project is fundamental for documentation and promotion.

- bumped shipkit and improved comments in the build.file so that it's easier to understand
- updated configuration, remove unnecessary stuff, improved logging so that users will see useful info
- updated travis config for latest shipkit, added some missing travis configuration so that Travis is faster
- described thoroughly the Shipkit test drive scenario in readme file, refreshed readme

### CODE REVIEW

**Comment by mockitoguy** [comment]
Friendly ping!

File: README.md [hunk 1]
```diff
@@ -4,6 +4,42 @@ This project does not build currently. It is in progress.
...
+```
+git checkout -b shipkit-testing
+echo "Shipkit testing" >> README.md
+git commit README.md -m "Shipkit testing"
+git push --set-upstream origin shipkit-testing
+```
+
+3. **Pull request**: in GitHub, open PR with the dummy change pushed in the previous step.
+**Important**: set the "base fork" to your own fork because GitHub by default suggests upstream repo.
+Upstream repo is no good because you don't have permissions and you cannot merge immediately.
```
Comment (wwilk) [inline README.md:22]: This parameter in pull request is called "base" not "base fork"

**Review by wwilk — CHANGES_REQUESTED** [review]
Just a small change, thanks for updating it!

File: README.md (hunk 1 above)
Comment (mockitoguy) [reply README.md:22]: In GitHub Web UI it's called "base fork" or "base repository". This instruction step is for using GitHub Web and I will accompany it with a sweet screenshot :)

I'm moving on merging if that's ok, we can discuss / update the naming in the next PR.

**Pull request merged**